src/z_extra.js

```diff
@@ -169,6 +170,7 @@ else{
 	global.ScarletsVideoStreamer = ScarletsVideoStreamer;
 	global.ScarletsMediaPlayer = ScarletsMediaPlayer;
 	global.ScarletsMediaPresenter = ScarletsMediaPresenter;
+	global.MediaBuffer = MediaBuffer;
```

Comment:
Can you change this line to `global.ScarletsMediaBuffer = MediaBuffer;` ?

because it would add `MediaBuffer` as a global variable in `window.MediaBuffer`, we will need to add a prefix to avoid the `MediaBuffer` name conflicting with other variable on the developer/user side.